assets/js/components/SettingsNotice.js

```diff
@@ -22,30 +22,112 @@
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
+/**
+ * Internal dependencies
+ */
+import InfoIcon from '../../svg/info-icon.svg';
+import SuggestionIcon from '../../svg/suggestion-icon.svg';
+import WarningIcon from '../../svg/warning-icon.svg';
+
 export const TYPE_WARNING = 'warning';
 export const TYPE_INFO = 'info';
 export const TYPE_SUGGESTION = 'suggestion';
 
-export default function SettingsNotice( { children, type } ) {
+const getIconFromType = ( type ) => {
+	switch ( type ) {
+		case 'warning':
+			return <WarningIcon />;
+		case 'info':
+			return <InfoIcon />;
+		case 'suggestion':
+			return <SuggestionIcon />;
+		default:
+			return null;
+	}
+};
+
+const SettingsNoticeSingleRow = ( {
+	notice,
+	type,
+	Icon,
+	LearnMore,
+} ) => {
 	return (
-		<div
-			className={ classnames(
-				'googlesitekit-settings-notice',
-				`googlesitekit-settings-notice--${ type }`
-			) }
-		>
+		<div className="googlesitekit-settings-notice__row">
+			<div className="googlesitekit-settings-notice__icon">
+				{ Icon ? <Icon /> : getIconFromType( type ) }
+			</div>
 			<div className="googlesitekit-settings-notice__text">
-				{ children }
+				{ notice }
 			</div>
+			{ LearnMore && (
+				<div>
+					<LearnMore />
+				</div>
+			) }
+		</div>
+	);
+};
+
+const SettingsNoticeMultiRow = ( {
+	notice,
+	type,
+	Icon,
+	LearnMore,
+	children,
+} ) => {
+	return (
```

Comment:
As per discussion on ticket https://github.com/google/site-kit-wp/issues/3251#issuecomment-854900234
Rendering differently when `children` and `notice` are provided